src/pages/[platform]/start/manual-installation/index.mdx

```diff
@@ -118,7 +118,14 @@ Or define your data resource:
 ```ts title="amplify/data/resource.ts"
 import { a, defineData, type ClientSchema } from '@aws-amplify/backend';
 
-const schema = a.schema({});
+// The schema cannot be empty.
```

Comment:
Debatable whether this note is _entirely_ necessary, but since the example schema was originally empty, it felt worthy of an explicit note to me.

Reply:
I'd drop the comment. I think just having an explicit example in there now is sufficient

Reply:
Done.